plugin.d.ts

```diff
@@ -112,16 +112,28 @@ interface Signer {
   };
 }
 
-declare const signerFactory: Signer;
-declare const sign: (value: string, secret: string) => string;
-declare const unsign: (input: string, secret: string) => string | false;
-
 export interface FastifyCookieOptions {
   secret?: string | string[] | Signer;
   parseOptions?: CookieSerializeOptions;
 }
 
-declare const fastifyCookie: FastifyPluginCallback<NonNullable<FastifyCookieOptions>>;
+export interface Sign {
```

Comment:
Can you change it to `type` ?
```ts
export type Sign = (value: string, secret: string): string;
```

Reply:
instead of 
```js
export interface Sign {
  (value: string, secret: string): string;
}
```

export 

```js
export type Sign = (value: string, secret: string): string;
```

Did I understand correctly?

Reply:
Yes, there is no meaning to use `interface` if it is only a `function`.
Same apply to `Unsign` and `SignerFactory`.

Reply:
ok